src/data/reference-data/ReferenceDataImporter.js

```diff
@@ -46,7 +52,7 @@ export const ReferenceDataImporter = ({ onSuccess, onDismiss, namespace, name })
         placeholder: 'Select data',
         value: selectedReference,
         onChange: ({ value }) => setSelectedReference(value),
-        options: _.keys(ReferenceData),
+        options: referenceAliases,
```

Comment:
To accomplish that, instead of passing only reference labels as options to Select, let's separate the `value` and `label` for each option.

```ts
options: _.keys(ReferenceData).map((referenceName) => ({ value: referenceName, label: ... }),
```